pkg/test/framework/components/istio/ingress/interface.go

```diff
@@ -83,7 +83,7 @@ type Instance interface {
 	TCPAddress() net.TCPAddr
 	// DiscoveryAddress returns the external XDS (!5012) address on the ingress gateway (or the NodePort address,
 	// when in an evnironment that doesn't support LoadBalancer).
```

Comment:
This would need to be updated, although it's not clear to me that the function should be overloaded to accept a service name like this. The struct represents an ingress gateway instance, so it seems like it shouldn't be used to get the address for a remote istiod

Reply:
fixed